### PR TITLE
[py.test/TestbedProcessing] Fixes for py.test

### PR DESCRIPTION
**Pull request by msosyak**
### Description of PR

Tried to run test_bgp_speaker and faces some issues

- missed "ptf" in  testbed
- missed --module-path= in pytest_cmd
- inventory_file contains absolute path not relative
- dut.host.options["inventory_manager"].get_vars failed with undefined method

Summary:
Fixes # (issue)

### Type of change

- [x] Bug fix
- [] Testbed and Framework(new/improvement)
- [] Test case(new/improvement)

#### How did you do it?
Fixed

-  TestbedProcessing.py to add "ptf" column  to testbed.csv
- Fixed pytest_cmd generation in pytest_runner.yml
- Replace inventory_manager.get_vars according to https://github.com/ansible/ansible/issues/33684

#### How did you verify/test it?
Run a few py.test testcases

### CODE REVIEW

File: ansible/TestbedProcessing.py [hunk 1]
```diff
@@ -213,11 +213,14 @@ def makeTestbed(data, outfile):
...
                 vm_base = groupDetails.get("vm_base")
                 dut = groupDetails.get("dut")
+                ptf = ""
                 comment = groupDetails.get("comment")
 
                 # catch empty types
                 if not groupName:
                     groupName = ""
+                else:
+                    ptf = "ptf_" + groupName
```
Comment (stepanblyschak) [inline ansible/TestbedProcessing.py]: according to this file https://github.com/Azure/sonic-mgmt/blob/master/tests/vtestbed.csv
ptf entry is not "ptf_" + groupName.
As I remember "ptf" column has to be a host name defined in inventory

File: tests/conftest.py [hunk 2]
```diff
@@ -89,7 +89,7 @@ def testbed_devices(ansible_adhoc, testbed):
         # when no ptf defined in testbed.csv
         # try to parse it from inventory
         dut = devices["dut"]
-        ptf_host = dut.host.options["inventory_manager"].get_vars(dut.hostname)["ptf_host"]
+        ptf_host = dut.host.options["inventory_manager"].get_host(dut.hostname).get_vars()["ptf_host"]
```
Comment (stepanblyschak) [inline tests/conftest.py]: According to PR description it looks like an bug in new ansible.
Could you put a comment saying that this is changed according to W/A descirbed in https://github.com/ansible/ansible/issues/33684?

Comment (msosyak) [reply tests/conftest.py]: This is not a W/A. `get_vars` was removed from the inventory manager.

https://github.com/ansible/ansible/commit/cea7151a5ba1a5339e828534ddd6000347867dea#diff-eaa1d70fa6781d189e1c19b20913f517

File: ansible/roles/test/tasks/pytest_runner.yml [hunk 3]
```diff
@@ -45,6 +45,10 @@
   set_fact:
     pytest_cmd: '{{ pytest_cmd }} --host-pattern={{ testbed_name }}'
 
+- name: append module path
+  set_fact:
+    pytest_cmd: '{{ pytest_cmd }} --module-path=../ansible/library'
```
[the remark this answers is not in the document]
Comment (msosyak) [reply ansible/roles/test/tasks/pytest_runner.yml]: I was run py.test command manually. Missed this in pytest_runner

[the remark this answers is not in the document]
Comment (msosyak) [reply ansible/roles/test/tasks/pytest_runner.yml]: But I would suggest leaving this for debugging purposes.

[the remark this answers is not in the document]
Comment (stepanblyschak) [reply ansible/roles/test/tasks/pytest_runner.yml]: I see, so you would like to have "--module-path" in command string so you can copy paste to run py.test mannually

**Pull request merged**